src/main/java/io/kokuwa/maven/helm/CleanMojo.java

```diff
@@ -0,0 +1,82 @@
+package io.kokuwa.maven.helm;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Comparator;
+import java.util.stream.Stream;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+
+/**
+ * Cleanups helm specific directory {@code charts} and {@code Chart.lock} file.
+ *
+ * @author Slawomir Jaranowski
+ * @since 6.16.1
```

Comment:
@sschnabe - Now I starting think of next releases - if should be minor version bump, so 6.17.0 as we have some of new feature. 

Up to you.